src/processor/queue-consumer.service.ts

```diff
@@ -177,11 +311,11 @@ export class QueueConsumerService extends WorkerHost implements OnApplicationBoo
       let outOfCapacity = remainingCapacity <= 0n;
 
       if (!outOfCapacity) {
-        this.logger.debug(`Capacity remaining: ${remainingCapacity}`);
+        // this.logger.debug(`Capacity remaining: ${remainingCapacity}`);
         if (capacityLimit.type === 'percentage') {
           const capacityLimitPercentage = BigInt(capacityLimit.value);
           const capacityLimitThreshold = (capacity.totalCapacityIssued * capacityLimitPercentage) / 100n;
-          this.logger.debug(`Capacity limit threshold: ${capacityLimitThreshold}`);
+          // this.logger.debug(`Capacity limit threshold: ${capacityLimitThreshold}`);
```

Comment:
Uncomment?

Reply:
Ugh. These are SO noisy... I know they're `.debug`, but it was killing me... maybe I'll consolidate them a bit so there are not so many lines spitting out...

Reply:
I've put in a solution that keeps the noise to a minimum, but still logs when relevant